waypoint_manager/package.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>waypoint_manager</name>
+  <version>0.0.0</version>
+  <description>Waypoint manager package</description>
+
+  <maintainer email="ajshank@noreply.users.github.com">ajshank</maintainer>
+  <author email="ajshank@noreply.users.github.com">ajshank</author>
+
```

Comment:
Add another <author> tag here.

Reply:
I assume you mean as `maintainer` and `author`, done.